lua/git-worktree/init.lua

```diff
@@ -198,17 +221,18 @@ end
 
 M.create_worktree = function(path, upstream)
     status:reset(8)
+    status:status("Creating Worktree")
 
     if upstream == nil then
-        error("Please provide an upstream...")
+        status:error("Please provide an upstream...")
     end
 
     has_worktree(path, function(found)
         if found then
-            error("worktree already exists")
+            status:error("worktree already exists")
         end
 
-        has_branch(path, function(found_branch)
+        has_branch(upstream, function(found_branch)
```

Comment:
do we have to have the upstream? 

If my upstream is a branch that doesn't exist yet, is that a problem?

Reply:
that is why i am writing tests now, to check all the variants and make sure they are correctly configured

I want to check  no upstream, origin, and upstream(like using a fork)